common/thorhelper/thorsoapcall.cpp

```diff
@@ -2434,7 +2434,24 @@ class CWSCAsyncFor : implements IWSCAsyncFor, public CInterface, public CAsyncFo
         unsigned retryInterval = 0;
 
         Url &url = master->urlArray.item(idx);
-        createHttpRequest(url, request);
+        StringBuffer spanName("SoapCall ");
+        url.getUrlString(spanName);
+
+        Owned<ISpan> clientSpan;
+        ISpan * activeSpan = master->logctx.queryActiveSpan();
+        clientSpan.setown(activeSpan->createClientSpan(spanName.str()));
+        //master->logctx.setActiveSpan(clientSpan.get());
+
+        Owned<IProperties> clientHeaders = ::getClientHeaders(clientSpan);
+
+        clientSpan->getClientHeaders(clientHeaders.get());
+        //fprintf(stderr, "clientspan clientheader: %s\n", clientHeaders->queryProp("traceparent"));
+
+        createHttpRequest(url, request, clientHeaders.get());
+        clientSpan->setSpanAttribute("target_host", url.host.get());
```

Comment:
Should be "network.peer_address" - see https://opentelemetry.io/docs/specs/semconv/http/http-spans/